CatAnalyzer/test/customise_cff.py

```diff
@@ -0,0 +1,27 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
I have tested it as first option.
so, "create-batch" loads "process" in "run_TtbarBbbarDiLeptonAnalyzer_cfg.py" before loading filelist.
And, after (loading filelist) then, it loads "customise_cff.py" in "process".